trp_project_scrum/security/ir_rules_collaborators_followers.xml

```diff
@@ -0,0 +1,131 @@
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="us_rule_foll" model="ir.rule">
+            <field name="name">User Story for collaborators followers</field>
+            <field name="model_id" ref="project_scrum.model_project_scrum_us"/>
+            <field name="domain_force">[
+                ('project_id.message_follower_ids', 'in', [user.partner_id.id])]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <!-- if the model was different and the actors where mapped to res
+             users it would had been more logical we have to work on the members of
+             the project-->
+
+        <!-- allows to see the task if you are part of the  project
+             AND if it is assigned to you , so collavborators won't see other's tasks.-->
+
+        <record id="meeting_rule_foll" model="ir.rule">
+            <field name="name">Meeting for collaborators followers</field>
+            <field name="model_id" ref="project_scrum.model_project_scrum_meeting"/>
+
+            <field name="domain_force">[
+                ('project_id.message_follower_ids', 'in', [user.partner_id.id] )]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+
+        <record id="test_rule_foll" model="ir.rule">
+            <field name="name">Test Case for collaborators followers</field>
+            <field name="model_id" ref="project_scrum.model_project_scrum_test"/>
+            <field name="domain_force">[
+                ('project_id.message_follower_ids', 'in', [user.partner_id.id] )]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="True" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="actors_rule_foll" model="ir.rule">
```

Comment:
also here, `domain_force` is missing. and also here, i think we don't really need to restrict this